CHANGE: Enable HID/DualShock templates for UWP (WSA)

timkeo opened the pull request.

timkeo committed:
CHANGE: Enable HID/DualShock templates for UWP (WSA)

Rene-Damm reviewed (changes requested):
Looking good. Please add a test, though, that makes sure we're recognizing the DualShock as it looks on UWP. Something like Devices_SupportsDualShockOnUWP in DualShockTests.

timkeo committed:
CHANGE: Address PR feedback

Add HID device test sfor UWP/WSA which utilaizes differnt query logic
Enable XInput device tests for UWP/WSA
